app/javascript/article-form/articleForm.jsx

```diff
@@ -1,4 +1,3 @@
-import 'preact/devtools';
```

Comment:
I've removed `preact/devtools` as it no longer exists. `preact/debug` is what should be used, but it should only be for development purposes. I will address this in another PR.

See [Debugging Preact Apps](https://preactjs.com/guide/v10/debugging/#strip-devtools-from-production)

Reply:
Thanks for catching this!